.github/workflows/pull-request.yml

```
@@ -145,8 +162,24 @@ jobs:
         run: docker compose up --exit-code-from api
-name: Pull request workflow
-on:
-  pull_request:
-    branches:
-      - "**"
+
+  Test-Docusaurus-Deployment:
+    name: Test Deployment to https://docs-api.talawa.io
+    runs-on: ubuntu-latest
+    needs: [Run-Tests]
+    # Run only if the develop-postgres branch and not dependabot    
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)